Antidote/CallsManager.m

```diff
@@ -421,7 +435,7 @@ - (void)switchViewControllerForCall:(OCTCall *)call
 {
     AALogVerbose(@"%@", call);
 
-    if ([self.currentCallViewController isKindOfClass:[ActiveCallViewController class]]) {
+    if ([self isCurrentViewControllerReusableForCurrentCall:call]) {
```

Comment:
Otherwise the other option:
`if ([self.currentCallViewController isKindOfClass:[ActiveCallViewController class] && (call.status == OCTCallStatusActive || call.status == OCTCallStatusPaused)) `

Which is quite messy